torchao/quantization/autoquant.py

```diff
@@ -194,15 +195,16 @@ def __torch_function__(cls, func, types, args=(), kwargs=None):
             )
             cls.log_shape(mat1, w_autoquant, bias)
             return func(mat1, w_autoquant.weight, bias)
-        try:
-            with torch._C.DisableTorchFunctionSubclass():
-                return func(*args, **kwargs)
-        except:
-            print(f"ERR: subclass doesn't implement {func}")
```

Comment:
problem is this just returns None I think, although it might be needed to implement `not error_on_unseen`, we need to think about how to do that if we change this